KsApi/models/graphql/adapters/Project+FetchAddOnsQueryDataTests.swift

```diff
@@ -9,6 +9,7 @@ final class Project_FetchAddOnsQueryDataTests: XCTestCase {
       XCTAssertNotNil(fragment)
 
       let project = Project.project(from: fragment)
+      XCTAssertEqual(project?.memberData.permissions.count, 6)
```

Comment:
Not sure if this makes sense including in here but think it's a good idea to add little tests like this any time we extend a Fragment. What do you think?

Reply:
Yea I did the same for `personalization`